atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -133,8 +134,10 @@ def get_pullspecs(self, digests):
                         "registry": registry.uri,
                         "repository": image.to_str(registry=False, tag=False),
                         "tag": image.tag or 'latest',
-                        "digest": digests[image.to_str()]
+                        "digest": digests[image.to_str()].default,
+                        "version": get_manifest_media_version(digests[image.to_str()])
```

Comment:
Can we cache `digests[image.to_str()]` in a local variable?

Reply:
Fixed